boards/iotlab-common/Makefile.include

```diff
@@ -10,6 +10,18 @@ PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.usbserial*B)))
 export BAUD = 500000
 include $(RIOTMAKE)/tools/serial.inc.mk
 
+# export ROM partitions
+export ROM_SIZE = 0x80000
```

Comment:
They're used in several places not only in Makefiles but also in flashing scripts and in subsequent linking operations, but I can modify this to export only those really needed. Does it harm?